clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -409,6 +410,19 @@ interface ClearlyDefinedService {
     @POST("definitions")
     fun getDefinitions(@Body coordinates: Collection<String>): Call<Map<String, Defined>>
 
+    /**
+     * Search for existing definitions based on the [pattern] string provided. The pattern string should contain
+     * the parts of the coordinates (typically namespace, name, and version) relevant for the search. Result is a
+     * list with the ClearlyDefined URIs to all the definitions that are matched by the pattern.
+     *
+     * Note that the _/definitions_ GET endpoint behaves differently depending on the parameters passed in. If only
```

Comment:
Formatting nit: Maybe better `"definitions" endpoint`? Also: What "parameters" (plural)? The function only takes *one* "pattern" parameter. If you're referring to the (potentially multiple) coordinate components mentioned above, then maybe better say "depending on the coordinates passed in".

Reply:
Reworded and dropped the part which is not relevant to this function.